nebula-logger/core/main/configuration/objects/LoggerFieldMapping__mdt/fields/TargetField__c.field-meta.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>TargetField__c</fullName>
+    <externalId>false</externalId>
+    <fieldManageability>SubscriberControlled</fieldManageability>
+    <inlineHelpText
+    >The target field to populate with the value of the source field. The data type of this field must match the data type of the source field - mappings that use different/incompatible data types will be ignored, resulting in lost logging data.</inlineHelpText>
+    <label>Target Field</label>
+    <metadataRelationshipControllingField>LoggerFieldMapping__mdt.TargetSObjectType__c</metadataRelationshipControllingField>
+    <referenceTo>FieldDefinition</referenceTo>
+    <relationshipLabel>Logger Target Field Mappings</relationshipLabel>
+    <relationshipName>LoggerTargetFieldMappings</relationshipName>
+    <required>true</required>
+    <type>MetadataRelationship</type>
+    <unique>true</unique>
```

Comment:
After talking through this with @jamessimone some more, I'm going to stick with using `unique == true` here - I can't think of a valid situation where multiple mappings should exist for the same target field (since only one value ultimately can be saved to the field), so making this unique makes sense.